README.md

```diff
@@ -8,7 +8,7 @@
 Trippy combines the functionality of traceroute and ping and is designed to assist with the analysis of networking
 issues.
 
-<img src="assets/0.8.0/trippy.gif" alt="trippy"/>
+<img src="https://raw.githubusercontent.com/fujiapple852/trippy/master/assets/0.8.0/trippy.gif" alt="trippy"/>
```

Comment:
This has to be an absolute url as the assets (gifs) are not in the `trippy` crate any more.  I'm hopeful that this will be rendered correctly by crates.io / docs.rs / lib.rs etc.

Reply:
Yes not a blocker for merging, we can test separately and adjust if it doesn't work.